src/js/popup.js

```diff
@@ -1016,18 +1004,44 @@ Logic.registerPanel(P_CONTAINER_EDIT, {
 
     document.querySelector("#edit-container-panel-name-input").value = identity.name || "";
     document.querySelector("#edit-container-panel-usercontext-input").value = userContextId || NEW_CONTAINER_ID;
-    const containerName = document.querySelector("#edit-container-panel-name-input");
-    window.requestAnimationFrame(() => {
-      containerName.select();
-      containerName.focus();
-    });
     [...document.querySelectorAll("[name='container-color']")].forEach(colorInput => {
       colorInput.checked = colorInput.value === identity.color;
     });
     [...document.querySelectorAll("[name='container-icon']")].forEach(iconInput => {
       iconInput.checked = iconInput.value === identity.icon;
     });
 
+    // Clear the proxy field before doing the retrieval requests below
+    document.querySelector("#edit-container-panel-proxy").value = "";
+
+    const edit_proxy_dom = function(result) {
+      if(result.type === "http")
+        document.querySelector("#edit-container-panel-proxy").value = result.host.toString() + ":" + result.port.toString();
```

Comment:
I suspect this works the same and is a little more readable:
```suggestion
        document.querySelector("#edit-container-panel-proxy").value = `${result.host}:${result.port}`;
```